config/samples/e2e/machine-with-non-ephemeral-resource/README.md

```diff
@@ -0,0 +1,63 @@
+# Machine Sample deployement
+
+This example deploys a `Machine` with `non-ephemeral` `volume` and `networkinterface`. 
+The following artifacts will be deployed in your namespace:   
+- IronCore `Network`, `NetworkInterface` and `VirtualIP`
+- IronCore `Machine` 
+- IronCore `Volume`
+- Secret containing the `ignition`
+
+## Prerequisites
+
+- [Butane](https://coreos.github.io/butane/)
+
+## Usage
+1. Adapt the `namespace` in `kustomization.yaml`
+2. Replace `your-user` [^1], `your-pw-hash`  [^2] and `your-ssh-key` [^3] in the `ignition/ignition.yaml`
```

Comment:
I would remove the footnotes as it should be clear what to change given the keys.

Reply:
done